dtest_setup.py

```diff
@@ -72,6 +77,13 @@ def __init__(self, dtest_config=None, setup_overrides=None, cluster_name="test")
         self.create_cluster_func = None
         self.iterations = 0
 
+    @ignore_log_patterns.setter
```

Comment:
```
    @ignore_log_patterns.setter
NameError: name 'ignore_log_patterns' is not defined
```

blah, python docs said this should work!  guess I need a getter... will try that